bin/migrate-oats-data/sql/insert-batch-application.sql

```diff
@@ -26,63 +25,111 @@ WITH
+                    WHEN oats_gov.oats_gov_name LIKE 'Cariboo%' THEN 'Cariboo Regional District'
+                    WHEN oats_gov.oats_gov_name LIKE 'Fraser Valley%' THEN 'Fraser Valley Regional District'
+                    WHEN oats_gov.oats_gov_name LIKE 'Columbia Shuswap%' THEN 'Columbia Shuswap Regional District'
+                    WHEN oats_gov.oats_gov_name LIKE 'Central Okanagan%' THEN 'Central Okanagan Regional District'
+                    WHEN oats_gov.oats_gov_name LIKE 'Squamish Lillooet%' THEN 'Squamish Lillooet Regional District'
+                    WHEN oats_gov.oats_gov_name LIKE 'Alberni-Clayoquot%' THEN 'Alberni-Clayoquot Regional District'
+                    WHEN oats_gov.oats_gov_name LIKE 'qathet%' THEN 'qathet Regional District'
+                    WHEN oats_gov.oats_gov_name LIKE 'Peace River%' THEN 'Peace River Regional District'
+                    WHEN oats_gov.oats_gov_name LIKE 'Okanagan Similkameen%' THEN 'Okanagan Similkameen Regional District'
+                    WHEN oats_gov.oats_gov_name LIKE 'East Kootenay%' THEN 'East Kootenay Regional District'
+                    WHEN oats_gov.oats_gov_name LIKE 'Bulkley-Nechako%' THEN 'Bulkley-Nechako Regional District'
+                    WHEN oats_gov.oats_gov_name LIKE 'Sunshine Coast%' THEN 'Sunshine Coast Regional District'
+                    WHEN oats_gov.oats_gov_name LIKE 'Nanaimo%' THEN 'Nanaimo Regional District'
+                    WHEN oats_gov.oats_gov_name LIKE 'Kitimat Stikine%' THEN 'Kitimat Stikine Regional District'
+                    WHEN oats_gov.oats_gov_name LIKE 'North Okanagan%' THEN 'North Okanagan Regional District'
+                    WHEN oats_gov.oats_gov_name LIKE 'Fraser Fort George%' THEN 'Fraser Fort George Regional District'
+                    WHEN oats_gov.oats_gov_name LIKE 'Cowichan Valley%' THEN 'Cowichan Valley Regional District'
+                    WHEN oats_gov.oats_gov_name LIKE 'Kootenay Boundary%' THEN 'Kootenay Boundary Regional District'
+                    WHEN oats_gov.oats_gov_name LIKE 'Comox Valley%' THEN 'Comox Valley Regional District'
+                    WHEN oats_gov.oats_gov_name LIKE 'Central Kootenay%' THEN 'Central Kootenay Regional District'
+                    WHEN oats_gov.oats_gov_name LIKE 'Capital%' THEN 'Capital Regional District'
+                    WHEN oats_gov.oats_gov_name LIKE 'Metro Vancouver%' THEN 'Metro Vancouver Regional District'
+                    WHEN oats_gov.oats_gov_name LIKE 'Central Coast%' THEN 'Central Coast Regional District'
+                    WHEN oats_gov.oats_gov_name LIKE 'North Coast%' THEN 'North Coast Regional District'
+                    WHEN oats_gov.oats_gov_name LIKE 'Strathcona%' THEN 'Strathcona Regional District'
+                    WHEN oats_gov.oats_gov_name LIKE 'Mount Waddington%' THEN 'Mount Waddington Regional District'
+                    ELSE oats_gov.oats_gov_name
```

Comment:
it is better to have temp tables for 10+ 'case when' statements. No need to refactor now, only if you need to come back to this place.